packages/renderer/src/lib/container/ContainerDetailsTtyTerminal.spec.ts

```diff
@@ -67,15 +68,17 @@ test('expect being able to attach terminal ', async () => {
   // wait attachContainerMock is called
   await waitFor(() => expect(attachContainerMock).toHaveBeenCalled());
 
+  await new Promise(resolve => setTimeout(resolve, 1000));
+
   // write some data on the terminal
   onDataCallback(Buffer.from('hello\nworld'));
 
   // wait 1s
-  await new Promise(resolve => setTimeout(resolve, 1000));
+  await new Promise(resolve => setTimeout(resolve, 2500));
```

Comment:
need to change the comment on the line before as it's no longer 1s

also do we still need the await timeout if we're waiting for the line 80

Reply:
but then it seems we're expecting a content

so what if we're just waiting to have the right text content ?
```
 await new Promise(resolve => setTimeout(resolve, 2000));

  // check the content
  expect(terminalLinesLiveRegion).toHaveTextContent('hello world');
```

==>

```
await vi waitFor(() => expect(terminalLinesLiveRegion).toHaveTextContent('hello world');
```

and as default timeout for waitFor is 1s, 

=> 

```
await vi waitFor(() => expect(terminalLinesLiveRegion).toHaveTextContent('hello world', {timeout: 2500});
```